reference-implementation/lib/writable-stream.js

```diff
@@ -300,11 +300,7 @@ function WritableStreamFinishInFlightClose(stream) {
 }
 
 function WritableStreamFinishInFlightCloseInErroredState(stream) {
```

Comment:
I think on balance that the explanatory value they provide wins. They serve to highlight that we do the same thing on both code paths, which otherwise would be less obvious.